### PR TITLE
Add missing templates in adviser and use Investigator

### PR DESCRIPTION
**Pull request by pacospace**
Depends-On: https://github.com/thoth-station/thoth-application/pull/93

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Enforce convention on structure
- Correct indentation in workflow task
- Add missing task in adviser using investigator producer of messages for unresolved packages
- Remove Kafka env variables not required

### CODE REVIEW

File: core/base/imagestreams.yaml [hunk 1]
```diff
@@ -17,6 +17,22 @@ spec:
...
+metadata:
+  name: investigator
+spec:
+  lookupPolicy:
+    local: true
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: investigator
```
Comment (goern) [inline core/base/imagestreams.yaml:29]: this is double?!

File: core/overlays/test/imagestreamtags.yaml [hunk 2]
```diff
@@ -26,6 +26,34 @@ spec:
...
+          kind: DockerImage
+          name: "quay.io/thoth-station/investigator:v0.2.0"
+        importPolicy: {}
+        referencePolicy:
+          type: Local
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: investigator
```
Comment (goern) [inline core/overlays/test/imagestreamtags.yaml:47]: dup?

File: investigator/overlays/test/kustomization.yaml [hunk 3]
```diff
@@ -5,8 +5,6 @@ commonLabels:
 resources:
   - ../../base
   - thoth-notification.yaml
-patchesStrategicMerge:
```
Comment (goern) [inline investigator/overlays/test/kustomization.yaml:8]: why is it removed?

Comment (pacospace) [reply investigator/overlays/test/kustomization.yaml:8]: is it required? if the image is in core? what patch we need to apply here?

Comment (goern) [reply investigator/overlays/test/kustomization.yaml:8]: ja

**Pull request merged**